README.md

```diff
@@ -73,132 +47,59 @@ RustScans **only** job is to reduce the friction between finding open ports and
+RustScan is a modern take on the port scanner. Sleek & fast. All while providing extensive extendability to you.
 
-Note: sometimes you can double click the file to achieve the same result.
+Not to mention RustScan uses Adaptive Learning to improve itself over time, making it the best port scanner for **you**.
 
-## Docker :whale:
+## Speed
 
-Docker is the recommended way of installing RustScan. This is because:
+Speed is guranteed via RustScan. However, if you want to run a slow scan that stealth reasons that is possible too.
 
-- It has a high open file descriptor limit, which is one of the [main problems](https://github.com/RustScan/RustScan/issues/40) with RustScan. Now you don't have to fiddle around trying to understand your OS.
-- It works on all systems, regardless of OS. Even Windows, which we don't officially support.
-- The Docker image uses the latest build from Cargo, our main source-of-truth package. This means that you will always be using the latest version.
-- No need to install Rust, Cargo, or Nmap.
+Firstly, let's talk code.
 
-To install Docker, [follow their guide](https://docs.docker.com/engine/install/).
+We have tests that check to see if RustScan is significantly lower than the previous version. If it is, the continuous integration fails and we can't commit code to master unless we make it faster.
 
-**Once Docker is installed, you can either build your own image using the `Dockerfile` (alpine) provided in the repo, or alternatively, use the published Docker image like below (most convenient)**
+[HyperFine](https://github.com/sharkdp/hyperfine) is used to monitor RustScan's performance over time to answer the question "Are we getting faster? Are we getting slower?".
 
-Please see our [DockerHub](https://hub.docker.com/repository/docker/rustscan/rustscan) for further our published versions. However, we recommend using our latest **major** release [1.10.0](https://github.com/RustScan/RustScan/releases/tag/1.10.0)
+Every pull request is reviewed by 1 person, but more often than not 2 people review it. We test it manually and make sure the code doesn't effect performance negatively.
 
-```
-Stable and supported: rustscan/rustscan:1.10.0
+[Read more here](https://github.com/RustScan/RustScan/wiki/Increasing-Speed-&-Accuracy).
 
-Bleeding edge (run at your own risk!): rustscan/rustscan:latest
-```
+## ⚙️ Extensible 
 
-We strongly recommend using the `1.10.0` tag, as this is the latest major - stable - release of RustScan. This README uses the `1.10.0` image by default, however, note that the `latest` image is considered experimental. You can use all releases of Docker by visiting the [DockerHub Tags](https://hub.docker.com/r/rustscan/rustscan/tags) and replacing the command with the tag you desire. i.e. `docker pull rustscan/rustscan:1.10.0` can be `docker pull rustscan/rustscan:1.6.0`
+RustScan has a new scripting engine which allows anyone to write scripts in most languages. Python, Lua, Shell are all supported.
```

Comment:
We don't have parsing built in for lua scripts, but that can be the next step to get it into the supported formats. On the other hand we do have perl script support already! And of course with some tricks in the execution format you can get lua scripts run as well. Python, Perl, Bash, or any kind of system installer program execution is supported. So basically they can build a script in a txt file that points to the lua script and the right interpreter, passing in only the ip and open ports.

Reply:
Thanks! I wrote this because we don't have documentation on this _yet_ and I was trying to think of what you did :sweat_smile: I'll change it to include this :D